doctr/datasets/vocabs.py

```diff
@@ -13,7 +13,7 @@
     "digits": string.digits,
     "ascii_letters": string.ascii_letters,
     "punctuation": string.punctuation,
-    "currency": "£€¥¢฿",
+    "currency": "£€¥¢฿₪",
```

Comment:
@uriva This needs to be added to Line 47 otherwise it would break our existing models :)